src/RepoAuditor/Plugins/GitHub/ClassicBranchProtectionRequirements/Impl/ClassicValueRequirementImpl.py

```diff
@@ -59,3 +59,6 @@ def __init__(
             requires_explicit_include=requires_explicit_include,
             missing_value_is_warning=False,
         )
+
+        # Reassign variable as semantic sugar.
+        self.github_settings_value = self.github_value

```

Comment:
I'm not sure why this sugar is necessary or how it is desirable to have 2 attributes that reference the same thing. If this is absolutely necessary, please add more context explaining why.

Reply:
So the tests for the subclasses of `ClassicValueRequirementImpl` use `requirement.github_settings_value` but I can see how this is pointless. I'll remove it.